src/main/resources/application.conf

```diff
@@ -12,7 +12,7 @@ com.ram.batch {
 
     mongodb {
         uri="mongodb://%s:%s@%s/?authSource=admin&readPreference=primary&ssl=false&directConnection=true"
-        host="localhost:27017"
+        host="ec2-54-89-66-18.compute-1.amazonaws.com:27017"
```

Comment:
Don't put your local change to configuration onto github. Especially when it contains secrets